### PR TITLE
Chem Grenades can be made with any reagent container.

### PR DESCRIPTION
**Pull request by Irradiation**
Realism.

capture

[Honk][tweak][qol]

:cl:
 * tweak: Chem grenades can now be made with any reagent container.

### CODE REVIEW

**Comment by yclatious** [comment]
Blessed.
Based.
Briliant.
Bitchin.

**Comment by SonixApache** [comment]
you realize that includes food, syringes, pills and bloodpacks, right?

**Comment by bathosbathtime** [comment]
You say that like it is a bad thing.
Botany rise up

**Comment by Irradiation** [comment]
@SonixApache *Yes*.

File: code/game/objects/items/weapons/grenades/chem_grenade.dm [hunk 1]
```diff
@@ -106,7 +106,7 @@
 				var/temp_reagents = new/list()
 				var/reagents_text = ""
 				for(var/obj/item/weapon/reagent_containers/G in beakers)
-					if(istype(G, /obj/item/weapon/reagent_containers/glass/beaker) || istype(G, /obj/item/weapon/reagent_containers/glass/bottle) || istype(G, /obj/item/weapon/reagent_containers/food/drinks))
+					if(istype(G, /obj/item/weapon/reagent_containers/glass/beaker))
```
Comment (Intigracy) [inline code/game/objects/items/weapons/grenades/chem_grenade.dm]: this removes adminlogging

**Comment by Intigracy** [comment]
This should really just be restricted to anything with OPENCONTAINER

**Comment by SonixApache** [comment]
I think OPENCONTAINER would work better than something as silly as spaghetti grenades
just saying

File: code/game/objects/items/weapons/grenades/chem_grenade.dm (hunk 1 above)
Comment (Irradiation) [reply code/game/objects/items/weapons/grenades/chem_grenade.dm]: Huh, I forgot to change this part around.

**Comment by Intigracy** [comment]
All /food items have `volume = 50` for reagents
but food/snack items have `volume = 100` for reagents, equivilant to large beakers

you know what's a food/snack?
cheese wedges, cut from a cheese wheel

for whatever reason, ice cream cones have `volume = 500`

**Comment by bathosbathtime** [comment]
I welcome the ice cream meta

File: code/game/objects/items/weapons/grenades/chem_grenade.dm (hunk 1 above)
Comment (Irradiation) [reply code/game/objects/items/weapons/grenades/chem_grenade.dm]: Solved (TM)

**Comment by Intigracy** [comment]
I have been notified that this is a PR intended for April 1, and as such is fine as-is.

**Comment by SonixApache** [comment]
we've got april fools override code just for that

**Comment by Intigracy** [comment]
If you want this to stay in after april fools then make another pr that edits the New() to change the lists based on if it's april fools or not

**Pull request merged**